gpt4all-chat/modellist.cpp

```diff
@@ -1110,7 +1110,7 @@ void ModelList::parseModelsJsonFile(const QByteArray &jsonData, bool save)
         if (!file.open(QIODeviceBase::WriteOnly)) {
             qWarning() << "ERROR: Couldn't write models config file: " << modelsConfig;
         } else {
-            file.write(jsonData.constData());
```

Comment:
Yeah, i'm not sure why you're changing the debug output or things like this.

Reply:
`constData()` is not needed here because QFile::write has native support for QByteArray.

If you'd like, I can revert [`50bce22` (#2034)](https://github.com/nomic-ai/gpt4all/pull/2034/commits/50bce22f5fb2cf9a382cd4445dc8c557b17e31e9) and apply it as a separate PR. These are just little mistakes I noticed while I was working on this PR that I figured I should fix.

Reply:
Please keep the qPrintable and remove the constData if you wish.